
Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-lz565[0m
{"level":"warn","ts":1580392873.8223329,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580392873.8229597,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-sqg92[0m
{"level":"warn","ts":1580392874.7762408,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392874.778521,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-nzbf4[0m
{"level":"warn","ts":1580392880.9240844,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392880.9261622,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1369-li-j6hkk[0m
{"level":"warn","ts":1580392881.2196045,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580392884.6078618,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:b5681d938e8cd6a20a1ff075997c66850836f6fc,1369:e73021331f6aed08db661e09ef32c65d7ff514b4
DEBUG: ran git fetch --unshallow origin e73021331f6aed08db661e09ef32c65d7ff514b4: b5681d938e8cd6a20a1ff075997c66850836f6fc: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard b5681d938e8cd6a20a1ff075997c66850836f6fc in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge e73021331f6aed08db661e09ef32c65d7ff514b4 in 
Merged SHA e73021331f6aed08db661e09ef32c65d7ff514b4 with commit message 'cleanup elements processing' into base branch master
Merged SHA a3f0ecd817f5b8913ad7ec9180d09db5e40e898e with commit message 'handle concurrency' into base branch master
Merged SHA ce432700d09b9744c78a1f0b4f52ae94a975553b with commit message 'small steps towards making updates idempotent' into base branch master
Merged SHA 58d4faffce36edeef592647660942263c85d73ae with commit message 'can link a req to resp by id but needs more work' into base branch master
Merged SHA 84c51508ff006ac939e2e2b9f50bcc6165a3c264 with commit message 'initial changes towards new request logging' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1369
+ BUILD_NUMBER=12
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Thu Jan 30 14:01:31 UTC 2020
The logs for [lint] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1369/12.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1369 --build=12'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1369 '--comment=Thu Jan 30 14:01:31 UTC 2020
The logs for [lint] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1369/12.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1369 --build=12'

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1369-li-wc68w[0m
{"level":"warn","ts":1580392894.1588798,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392894.2112486,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1369-li-wt5jb[0m
{"level":"warn","ts":1580392894.4689116,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392894.790046,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-xm5ft[0m
{"level":"warn","ts":1580392898.981516,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580392898.9822252,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-72qlf[0m
{"level":"warn","ts":1580392899.9466066,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392899.9492774,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-4g7nk[0m
{"level":"warn","ts":1580392905.2293255,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392905.2312932,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-qf6mm[0m
{"level":"warn","ts":1580392905.6270344,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580392917.8755715,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-z7rgx-12[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading https://files.pythonhosted.org/packages/30/62/cf549544a5fe990bbaeca21e9c419501b2de7a701ab0afb377bc81676600/black-19.3b0-py36-none-any.whl (89kB)
Collecting click>=6.5
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
Collecting toml>=0.9.4
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting appdirs
  Downloading https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Collecting attrs>=18.1.0
  Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Installing collected packages: click, toml, appdirs, attrs, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
53 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
